packages/bpk-component-skip-link/examples.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import React from 'react';
+
+import BpkSkipLink from './index';
+
+const DefaultExample = () => (
+  <BpkSkipLink
+    style={{ position: 'absolute' }}
+    href="#main"
+    label="Skip to main content"
+  />
+);
```

Comment:
We should have another example to jump to #implementation for the docs page

Reply:
Ah good shout!

Reply:
Ooo good shout!